src/resources/objects/interfaces.ts

```diff
@@ -40,7 +40,9 @@ export interface ListObjectOptions extends PaginationOptions {
   name?: string;
 }
 
-export interface ListObjectSubscriptionsOptions extends PaginationOptions {}
+export interface ListObjectSubscriptionsOptions extends PaginationOptions {
```

Comment:
I don't think this is correct, because this type is shared between the `listSubscriptions` and `getSubscriptions` methods (param only applies to `listSubscriptions`)

Reply:
ah okay thanks for catching will update